### DISCUSSION TITLE
Wich "Host" should u use in the CumulocityConnectorConfig.json file?

### DISCUSSION THREAD

**Discussion by DriesDebouver** [hms-networks/flexy-cumulocity-connector#16] [Q&A] [answered]
In the QUICK-START.md file is the following:

![image](https://user-images.githubusercontent.com/28297720/187918459-8dd47d54-947d-432a-81a3-4f208376dde3.png)

Wich "Host" should u use in the _CumulocityConnectorConfig.json_ file?

Does Cumulocity provide a Host (MQTT Broker?) or should we use our own?

**Comment by alexjhawk** [comment, marked as answer]
Hi @DriesDebouver,

The Cumulocity cloud provides an MQTT broker host, so you should not need to provide your own.

The `Host` value should simply match the Cumulocity MQTT URL for your region/instance. For example, `mqtt.eu-latest.cumulocity.com` or `mqtt.us-latest.cumulocity.com`

There is some additional technical information from Cumulocity available at https://cumulocity.com/guides/device-sdk/mqtt-examples/#hello-mqtt, but in short, we only support the `mqtt.*.cumulocity.com` format for your particular instance.

Best regards,

@alexjhawk
